java3lesson6/src/main/java/FirstArrTask.java

```diff
@@ -0,0 +1,16 @@
+public class FirstArrTask {
+
+    public int[] arrCrop(int[] arr) {
+        int lastIndex = -1;
+        int[] newArr;
+        for (int i = 0; i < arr.length; i++) {
+            if(arr[i] == 4) lastIndex = i;
+        }
```

Comment:
```suggestion
        for (int i = 0; i < arr.length; i++) {
            if(arr[i] == 4) {
                lastIndex = i;
                break;
            }
            
        }
```
чтобы не делать лишние итерации